sound/soc/codecs/max98373.h

```diff
@@ -1,5 +1,6 @@
-// SPDX-License-Identifier: GPL-2.0
-// Copyright (c) 2017, Maxim Integrated
+/* SPDX-License-Identifier: GPL-2.0-only
```

Comment:
the ask is that the first line is in one comment, then a second comment for copyright, etc.

Reply:
Let me correct it as followed. Thanks.
/* SPDX-License-Identifier: GPL-2.0-only */
/* Copyright (c) 2017 Maxim Integrated */